src/Rules/UseSafeFunctionsRule.php

```diff
@@ -55,6 +55,16 @@ public function processNode(Node $node, Scope $scope): array
                         }
                     }
                 }
+                if ($functionName === "json_decode" || $functionName === "json_encode") {
+                    foreach ($node->args as $arg) {
+                        // Named argument might be not place on exact "documented" position
```

Comment:
```suggestion
```

I think this comment does not add much value, I actually find it rather confusing to find it in this exact position. If anything, this whole section could be prefaced with an explanation of checking for positional arguments + named arguments.